x-pack/plugins/security_solution/public/common/lib/cell_actions/default_cell_actions.tsx

```diff
@@ -34,13 +34,36 @@ const useKibanaServices = () => {
   return { timelines, filterManager };
 };
 
+/**
+ * rowIndex is bigger than `data.length` for pages with page numbers bigger than one.
+ * For that reason, we must calculate `rowIndex % itemsPerPage`.
+ *
+ * Ex:
+ * Given `rowIndex` is `13` and `itemsPerPage` is `10`.
+ * It means that the `activePage` is `2` and the `pageRowIndex` is `3`
+ *
+ * **Warning**:
+ * Be careful with array out of bounds. `pageRowIndex` can be bigger or equal to `data.length`
+ *  in the scenario where the user changes the event status (Open, Acknowledged, Closed).
+ */
+export const getPageRowIndex = (rowIndex: number, itemsPerPage: number) => rowIndex % itemsPerPage;
```

Comment:
This function is defined twice (timelines and security solutions). Where can we move it to reuse on both places?